packages/backend/src/yeomanui.ts

```diff
@@ -112,13 +112,13 @@ export class YeomanUI {
   }
 
   public async _notifyGeneratorsInstall(args: any[], force: boolean) {
+    this.uiOptions.installGens = _.isObject(args) && _.isEmpty(args) ? undefined : args;
```

Comment:
according to the signature type of `args` is `any[]`. is indeed, the type of `object` expected here?

Reply:
I just moved this code from last line to first line of the function to prevent override of the `installGens` value by race condition of multiple calls